10_transformers-from-scratch.ipynb

```diff
@@ -193,6 +193,13 @@
     "#### Memory mapping"
```

Comment:
<p>I suggest being explicit in what needs to be changed, e.g.</p><p> </p><blockquote><strong>Note</strong>: In the following code block, a large GPT-2 checkpoint is loaded into memory. On platforms like Colab and Kaggle, this can cause the instance to crash due to insufficient RAM or GPU memory. You can still run the example if you use the small checkpoint by replacing the configuration with <code>config = AutoConfig.from_pretrained("gpt2", vocab_size=len(tokenizer))<span class="ql-cursor">﻿</span></code></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nlp-with-transformers/notebooks/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='61'/>